homeassistant/components/emulated_roku/config_flow.py

```diff
@@ -0,0 +1,62 @@
+"""Config flow to configure emulated_roku component."""
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_NAME
+from homeassistant.core import callback
+from homeassistant.util import slugify
+from .const import (DOMAIN, CONF_LISTEN_PORT, CONF_HOST_IP,
+                    CONF_ADVERTISE_IP, CONF_ADVERTISE_PORT,
+                    CONF_UPNP_BIND_MULTICAST,
+                    DEFAULT_UPNP_BIND_MULTICAST)
+
+
+@callback
+def configured_servers(hass):
+    """Return a set of the configured servers."""
+    return set(slugify(entry.data[CONF_NAME]) for entry
```

Comment:
why slugify

Reply:
The components I've checked for config flow examples all used slugify for the name. Other than that, I don't see any reasons.

Reply:
drop it.